src/pages/RealEstateDetail/Deal/Deal.js

```diff
@@ -1,30 +1,38 @@
 import React, { useEffect } from 'react';
 import styled from 'styled-components';
-import KakaoMapScript from '../KakaoMapScript';
 
 import DealPoint from './DealWrap/DealPoint';
 import DealOutLine from './DealWrap/DealOutLine';
 import DealSecurity from './DealWrap/DealSecurity';
 import DealDetail from './DealWrap/DealDetail';
 import DealInfo from './DealWrap/DealInfo';
 
-function Deal() {
-  useEffect(() => {
-    KakaoMapScript();
-  }, []);
-
+function Deal(props) {
+  console.log(props.mortSeniorLoanAmount);
   return (
     <DealWrap>
       {/*투자포인트*/}
       <DealPoint />
       {/*상품개요*/}
       <DealOutLine />
       {/*담보안정성*/}
-      <DealSecurity />
+      <DealSecurity
+        mortSeniorLoanAmount={props.mortSeniorLoanAmount}
+        mortEstimatedRecovery={props.mortEstimatedRecovery}
+        dealInfoNetAmount={props.dealInfoNetAmount}
+      />
       {/*담보상세*/}
-      <DealDetail />
+      <DealDetail
+        mortgageInfoLatitude={props.mortgageInfoLatitude}
+        mortgageInfoLongitude={props.mortgageInfoLongitude}
+        mortgageInfoAddress={props.mortgageInfoAddress}
+        mortgageCompletedDate={props.mortgageCompletedDate}
+        mortgageScale={props.mortgageScale}
+        mortSupplyArea={props.mortSupplyArea}
+        mortFloor={props.mortFloor}
```

Comment:
props로 받은 모든 속성을 그대로 다시 전달해준다면 
```suggestion
      <DealDetail
         {...props}
```
로 포현할 수 있습니다